src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DES/DESCipherTests.cs

```diff
@@ -252,6 +253,236 @@ public static void EncryptWithLargeOutputBuffer(bool blockAlignedOutput)
+                key: "04d923abd9291c3e".HexToByteArray(),
+                iv: "191f8794944e601c".HexToByteArray(),
+                plainBytes: "6fe8f67d2af1".HexToByteArray(),
+                cipherBytes: "6012c9171bb8".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_6()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=6
+            // used only key1, cipherBytes computed using openssl
+            TestDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "a7799e7f5dfe54ce".HexToByteArray(),
+                iv: "370184c749d04a20".HexToByteArray(),
+                plainBytes: "2b4228b769795b".HexToByteArray(),
+                cipherBytes: "58d3de76687976".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_7()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=7
+            // used only key1, cipherBytes computed using openssl
+            TestDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "6bfe3d3df8c1e0d3".HexToByteArray(),
+                iv: "51e4c5c29e858da6".HexToByteArray(),
+                plainBytes: "4cb3554fd0b9ec82".HexToByteArray(),
+                cipherBytes: "16b3595259693776".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_8()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=8
+            // used only key1, cipherBytes computed using openssl
+            TestDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "e0264aec13e63db9".HexToByteArray(),
+                iv: "bd8795dba79930d6".HexToByteArray(),
+                plainBytes: "79068e2943f02914af".HexToByteArray(),
+                cipherBytes: "fe78cb95ce9e4cac2f".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_9()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=9
+            // used only key1, cipherBytes computed using openssl
+            TestDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "7ca28938ba6bec1f".HexToByteArray(),
+                iv: "953896586e49d38f".HexToByteArray(),
+                plainBytes: "2ea956d4a211db6859b7".HexToByteArray(),
+                cipherBytes: "81b850bf481db5df0437".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+        private static void TestDESTransformDirectKey(
```

Comment:
Missing blank line between methods

```suggestion

        private static void TestDESTransformDirectKey(
```